src/popup/Popup.tsx

```diff
@@ -15,9 +15,25 @@ export default function Popup(): JSX.Element {
     chrome.runtime.sendMessage({ popupMounted: true });
   };
 
+  const [isEnabled, setIsEnabled] = useState(true);
+
+  const changeAutolike = useCallback(() => {
+    setIsEnabled(!isEnabled);
+    chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) {
+      chrome.tabs.sendMessage(tabs[0].id, { isEnabled: isEnabled });
+    });
+  }, [isEnabled]);
+
   return (
     <div>
       Hello, {url}! <button onClick={sendAMessage}>yo</button>
+      <br></br>
```

Comment:
```suggestion
      <br/>
```
accomplishes the same thing